substrate/frame/utility/src/benchmarking.rs

```diff
@@ -19,7 +19,6 @@
 
 #![cfg(feature = "runtime-benchmarks")]
 
```

Comment:
```suggestion

use alloc::vec;
```

Reply:
Maybe deleting this was a mistake (?)

Reply:
I believe that `use crate::*` already included. This when I was working on this PR.

Reply:
So I applied this to the previous benchmarking that I have worked.

Reply:
It didn't compile for me without adding it back, the same on the CI too. I think that's because in the crate you have only `vec::Vec`.

Reply:
Hmm, this PR was opened 2 weeks ago. I might miss something.